index.html

```diff
@@ -12,9 +12,9 @@
       <button type="button" class="options-button">
         <img
           class="menu-button"
-          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Enabled.png"
+          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio\Icons\Enabled.png"
           alt="menu button"
-        >
+        />
```

Comment:
Let's start with your linter errors that are causing your checks not to pass:

You have the following WebHint errors that have something to do with your `img` tag `src` paths:
![linters](https://user-images.githubusercontent.com/38088818/164545638-c728a771-a141-45ef-a17e-b66cdcbd63f9.png)

For your `hamburger` button:
![image](https://user-images.githubusercontent.com/38088818/164545937-0d92a7d2-fad4-4483-95bf-ea6135c5ee21.png)

The correct syntax is:
```
      <button type="button" class="options-button">
        <img
          class="menu-button"
          src="./Icons/Enabled.png"
          alt="menu button"
        />
      </button>
```

This also solves your icon not displaying issue as well as seen below:
![image](https://user-images.githubusercontent.com/38088818/164546308-7cb32943-c4fb-4426-98c2-7f284332645e.png)